ui/appui-react/src/appui-react/icons/AssetClassificationHide.tsx

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module AssetClassificationHide
+ */
+
+import * as React from "react";
+export function AssetClassificationHide() {
```

Comment:
Spacing (applies to every icon file)

```suggestion
import * as React from "react";

export function AssetClassificationHide() {
```

Reply:
Fixed it everywhere